packages/mds-transactions-service/@types/index.ts

```diff
@@ -116,13 +116,11 @@ export interface TransactionService {
   addTransactionOperation: (operation: TransactionOperationDomainCreateModel) => TransactionOperationDomainModel
   // read back operations for a transaction
   // if auth token has a provider_id, it must match the provider_id in the transaction
-  getTransactionOperations: (
-    transaction_id: TransactionDomainModel['transaction_id']
-  ) => TransactionOperationDomainModel[]
+  getTransactionOperations: () => TransactionOperationDomainModel[]
 
   // get all the status changes for this transaction (typically we won't have a ton I expect)
   // if auth token has a provider_id, it must match the provider_id in the transaction
-  getTransactionStatuses: (transaction_id: TransactionDomainModel['transaction_id']) => TransactionStatusDomainModel[]
+  getTransactionStatuses: () => TransactionStatusDomainModel[]
```

Comment:
Same question for statuses. Always get all? I am also curious why we _add_ operations and _set_ statuses given that they both ultimately perform inserts and there are a list of each for every transaction. I assume order is important for statuses somehow? When you query a transaction do you want _all_ its operations and its _latest_ status?